include/nanorange/algorithm/minmax_element.hpp

```diff
@@ -10,12 +10,15 @@
 #ifndef NANORANGE_ALGORITHM_MINMAX_ELEMENT_HPP_INCLUDED
 #define NANORANGE_ALGORITHM_MINMAX_ELEMENT_HPP_INCLUDED
 
-#include <nanorange/algorithm/minmax.hpp>
+#include <nanorange/detail/algorithm/return_types.hpp>
+#include <nanorange/detail/functional/comparisons.hpp>
+#include <nanorange/detail/iterator/projected.hpp>
+#include <nanorange/detail/ranges/concepts.hpp>
```

Comment:
In the algorithms headers I usually just `#include <nanorange/ranges.hpp>` rather than individual `detail` headers -- it saves having to remember exactly what is defined in which header, and doesn't seem to affect compile times much (famous last words...)

Reply:
I was wondering about that, but I opted for the conservative approach since clangd was able to figure where things live once it had a compilation database.

Should `return_types.hpp` be added to some non-detail header, to make it implicitly included with the rest? Otherwise, I've made the requested changes.